### DISCUSSION TITLE
Importing coordinates to Relion

### DISCUSSION THREAD

**Discussion by AlkazaX** [tbepler/topaz#134] [Q&A]
Hello,

I’m struggling exporting particle coordinates from topaz to Relion, my topaz output is a single .star file that includes the micrograph name followed by x and y coordinates as this:
 
```
data_images
loop_
_rlnAutopickFigureOfMerit #1
_rlnMicrographName #2
_rlnCoordinateX #3
_rlnCoordinateY #4
7.248999        211007__doseweighted.mrc     848     2048
```
 
If I try to import this file as coordinate file and then use for extract particles it expects multiple coordinates.star files for each micrographs which is not the case so, it crashes because there is only one coordinates file. Using the file directly as input for Extract job does not fix the problem.
 
Could anybody indicate me how should I import this coordinate file for extracting the particles?

Thank you

Best
Victor

**Comment by tbepler** [comment]
The latest version of Topaz here on github (0.2.5 dev) has a flag in `topaz extract` called `--per-micrograph` to help with this. It will cause topaz to output a particle star file for each micrograph rather than one main particle file.
